src/uu/cp/src/platform/linux.rs

```diff
@@ -53,17 +76,105 @@ where
     match fallback {
         CloneFallback::Error => Err(std::io::Error::last_os_error()),
         CloneFallback::FSCopy => std::fs::copy(source, dest).map(|_| ()),
+        CloneFallback::SparseCopy => sparse_copy(source, dest),
+        CloneFallback::SparseCopyWithoutHole => sparse_copy_without_hole(source, dest),
+    }
+}
+
+#[cfg(any(target_os = "linux", target_os = "android"))]
+fn check_for_data(source: &Path) -> Result<(bool, u64, u64), std::io::Error> {
```

Comment:
could you please add a rustdoc to document this function?
bool, u64 and 64 as return values isn't obvious

Reply:
Yea, sure i will add some documentation for everything.